pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -419,8 +419,19 @@ type CommonServicePlanSpec struct {
 	// and its data will not be migrated.
 	//
 	// ServiceBindingCreateParameterSchema is the schema for the parameters that
-	// may be supplied binding to an ServiceInstance on this plan.
+	// may be supplied binding to a ServiceInstance on this plan.
 	ServiceBindingCreateParameterSchema *runtime.RawExtension `json:"serviceBindingCreateParameterSchema,omitempty"`
+
+	// Currently, this field is ALPHA: it may change or disappear at any time
+	// and its data will not be migrated.when a bind operation stored in the Secret when binding to a ServiceInstance on this plan.
+	//
+	// ServiceBindingCreateResponseSchema is the schema for the response that
+	// will be returned by the broker when binding to a ServiceInstance on this plan.
+	// The schema also contains the sub-schema for the credentials part of the
+	// broker's response, which allows clients to see what the credentials
+	// will look like even before the binding operation is performed.
+	// TODO: introduce new struct with both Parameter and Response schemas?
```

Comment:
Is this todo still valid?

Reply:
I'll remove it for now, but IMHO we should reorganize all the schema-related fields before moving them out of Alpha (they are currently all directly under `ServicePlanSpec`).